lib/app/routes.js

```diff
@@ -1,10 +1,18 @@
 'use strict';
 // app/routes.js
 module.exports = function(app, env) {
-  app.get('/', function(req, res) {
+  app.get('/hydraters', function(req, res) {
     res.render('index.ejs', {
       layout: false,
-    	env: env
+    	env: env,
+    	source: 'hydraters'
+    });
+  });
+  app.get('/providers', function(req, res) {
```

Comment:
res.render automatically give a response with 200 HTTP Status and Content-Type: text/html, calling next() after throws an error